config/model_configs/prognostic_edmfx_era5driven_column.yml

```diff
@@ -0,0 +1,50 @@
+initial_condition: "ERA5"
+external_forcing: "ERA5"
+external_forcing_file: "/central/groups/esm/jschmitt/era5/forcing/era5_monthly_forcing_1.nc"
+cfsite_number: "site23"
+surface_setup: "ERA5"
+turbconv: "prognostic_edmfx"
+implicit_diffusion: true
+implicit_sgs_advection: false
+approximate_linear_solve_iters: 2
+edmfx_upwinding: first_order
+rayleigh_sponge: true
+edmfx_entr_model: "PiGroups"
+edmfx_detr_model: "PiGroups"
+edmfx_sgs_mass_flux: true
+edmfx_sgs_diffusive_flux: true
+edmfx_nh_pressure: true
+edmfx_filter: true
+prognostic_tke: true
+moist: "equil"
+config: "column"
+z_max: 70e3
+truncation: 40000
```

Comment:
Can you document the "truncation" option or remove it from the config if it isn't used.